extra/resources/Dummy

```diff
@@ -199,6 +199,31 @@ dummy_monitor() {
     fi
     
     if [ -f "${OCF_RESKEY_state}" ]; then
+        # Multiple monitor levels are defined to support various tests
+        case "$OCF_CHECK_LEVEL" in
+            10)
+            # monitor level with delay, useful for testing timeouts
+            sleep 30
+            ;;
+
+            20)
+            # monitor level that fails intermittently
+            n=$(expr $(dd if=/dev/random bs=1 count=1 2>/dev/null | od | head -1 | cut -f2 -d' ') % 5)
```

Comment:
maybe /dev/urandom is good enough as /dev/random might run dry in a vm introducing unexpected delay here

Reply:
right, forgot which was which :)